core/templates/dev/head/css/oppia.css

```diff
@@ -858,6 +858,13 @@ textarea {
     margin-top: 0;
     margin-left: 5px;
   }
+  ul.nav.navbar-nav.oppia-navbar-nav.navbar-right {
+    display: flex;
+    float: right;
+    margin: 0;
+  }
+}
+@media(max-width: 651px) {
```

Comment:
Here's a link to screen capture at my end (latest commit): https://youtu.be/HCGsaph6a74